CHANGELOG.md

```diff
@@ -21,8 +21,11 @@
             if the output is a pipe or a file the escape sequences are stripped
             * Always - output methods **never** strip ANSI escape sequences, regardless of the output destination
             * Never - output methods strip all ANSI escape sequences
+    * Added ``macro`` command to create macros, which are similar to aliases, but can take arguments when called
+    * ``alias`` is now an argparse command with subcommands to create, list, and delete aliases 
```

Comment:
I think we should clarify that the alias command has changed syntax. All current alias commands in startup scripts or transcripts will break with this release.